src/components/Main/Main.tsx

```diff
@@ -76,12 +76,12 @@ async function runSimulation(
 
 const severityDefaults: SeverityTableRow[] = updateSeverityTable(severityData)
 
-const isCountry = (country: string): country is keyof typeof countryAgeDistribution => {
+const isCountry = (country: string): country is keyof typeof countryAgeDistributionData => {
   return countryAgeDistributionData.hasOwnProperty(country)
 }
 
-const isRegion = (region: string): region is keyof typeof countryCaseCounts => {
-  return countryCaseCounts.hasOwnProperty(region)
+const isRegion = (region: string): region is keyof typeof countryCaseCountData => {
+  return countryCaseCountData.hasOwnProperty(region)
 }
 
```

Comment:
Change `countryCaseCounts` to `countryCaseCountData` to match the renamed import.